Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -84,3 +85,85 @@ Deploy application
     cd "BlueprintBaseName.1/src/BlueprintBaseName.1"
     dotnet lambda deploy-serverless
 ```
+
+## Lambda Annotations
+This template now uses [Lambda Annotations](https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md) to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
+
+This automatically handles reading parameters from an APIGatewayProxyRequest and returning an APIGatewayProxyResponse. 
+
+It also generates the function resources in a JSON or YAML CloudFormation template based on your function definitions, and keeps them updated.
+
+### Using Annotations without API Gateway
+If you still wish to use Lambda Annotations for the CloudFormation template syncing without taking advantage of the integration with API Gateway, replace your function definition with the following:
+```
+    [LambdaFunction(PackageType = LambdaPackageType.Image, Policies = "AWSLambdaBasicExecutionRole", MemorySize = 256, Timeout = 30)]
+    public string Get(ILambdaContext context)
```

Comment:
This example still looks like an API Gateway based function just without the attribute. How about the following showing annotations with SQS.

```
[LambdaFunction(PackageType = LambdaPackageType.Image, Policies = "AWSLambdaBasicExecutionRole", MemorySize = 256, Timeout = 30)]
public async Task FunctionHandler(SQSEvent evnt, ILambdaContext context)
{
    foreach(var message in evnt.Records)
    {
        await ProcessMessageAsync(message, context);
    }
}
```

Reply:
Done, I did swap in `AWSLambdaSQSQueueExecutionRole` for the policy in this example